spec/runtime/with_unbundled_env_spec.rb

```diff
@@ -188,14 +179,14 @@ def build_bundler_context
     end
   end
 
-  describe "Bundler.clean_system", :bundler => "< 2" do
+  describe "Bundler.clean_system", :bundler => "< 3" do
```

Comment:
`Bundler.clean_system` should be tested on bundler 3.

Reply:
Marking as solved for the same reason as https://github.com/bundler/bundler/pull/6957#discussion_r264004133.